### DISCUSSION TITLE
How to  create or init an empty Line chart

### DISCUSSION THREAD

**Discussion by jianhaozh** [patrykandpatrick/vico#480] [Questions]
I have a view that needs to init an Empty line chart with grid lines, max Y value, and max X value,   but after I review the vico source code,  its hard for me to implement it,  can anybody show me a simple example, also , Giving an example about editing data value on line chart by drag the point up or down is also greatly appreciated.

**Comment by Gowsky** [comment]
Hello @jianhaozh, you can hide the line in the line chart by changing its color to a transparent one or its thickness to 0. As for editing chart values with gestures, that's something you'd need to build yourself.
